mitosheet/src/mito/components/endo/celleditor/CellEditorDropdown.tsx

```diff
@@ -210,6 +211,14 @@ const CellEditorDropdown = (props: {
                     </div>
                 </div>
             }
+            {/* Always display a link to the documentation */}
+            <a
+                className='text-body-2 text-body-2-link cell-editor-dropdown-link'
+                target='_blank'
+                href={FORMULA_DOC_LINK}
```

Comment:
Addressing this `Using target="_blank" without rel="noreferrer" (which implies rel="noopener") is a security risk in older browsers` should make the linter pass